### PR TITLE
Fix themed listing bug

### PR DESCRIPTION
**Pull request by okhanrt**

### CODE REVIEW

File: packages/react-ui-ag/src/Listings/__tests__/DesktopListing-test.js [hunk 1]
```diff
@@ -84,172 +83,114 @@ const props = {
...
+    {
+      children: 'Check Availability',
+      onClick: () => { },
+      className: 'contact',
+    },
+  ],
+}
 
-  it('fires onClick on card click', () => {
-    const cardClick = jest.fn()
```
[the remark this answers is not in the document]
Comment (okhanrt) [reply packages/react-ui-ag/src/Listings/__tests__/DesktopListing-test.js]: Not for `DesktopListing`. `Listing` component does this test so it's no longer needed.

[the remark this answers is not in the document]
Comment (mpelzsherman) [reply packages/react-ui-ag/src/Listings/__tests__/DesktopListing-test.js]: Thanks. It looks like a lot of props removed from `DesktopListing`; I take it that's all in `Listing` as well.

**Comment by mpelzsherman** [comment]
LGTM

File: packages/react-ui-ag/src/Listings/__tests__/DesktopListing-test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (okhanrt) [reply packages/react-ui-ag/src/Listings/__tests__/DesktopListing-test.js]: Yup!

**Comment by paudyalhari** [comment]
👍

**Pull request merged**